03_letters/CameronSchafer/letters.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+// string to rune with counts
+func letters(s string) map[rune]int {
+	tMap := make(map[rune]int)
```

Comment:
You could consider just having 't' as variable name, instead of including type in the variable. Same with tSlice/etc below.